framework/src/base/ComputeUserObjectsThread.C

```diff
@@ -73,23 +73,21 @@ void
 ComputeUserObjectsThread::onElement(const Elem * elem)
 {
   _fe_problem.prepare(elem, _tid);
```

Comment:
That is a good question. See the #8335 where Cody and I discussed. But I think we possibly can do something here. Also looks like `prepare` is not that expensive.